tests/scripts/tests-setup.sh

```diff
@@ -3,6 +3,16 @@
 
 set -x
 
+echo "Setting Host Network Backend as netavark..."
+sudo cp /usr/share/containers/containers.conf /etc/containers/containers.conf
```

Comment:
Are these `sudo` commands required?

Reply:
Yes they are but we changed the approach a bit in the new patch.